guides/rails_guides/markdown/renderer.rb

```diff
@@ -58,18 +66,20 @@ def convert_footnotes(text)
           end
         end
 
-        def class_for(code_type)
+        def lexer_language(code_type)
           case code_type
-          when "ruby", "sql", "plain", "js", "yaml"
+          when "js", "http", "html", "ruby", "sql", "yaml", "yml"
             code_type
           when "erb", "html+erb"
             "erb"
           when "html"
-            "xml" # HTML is understood, but there are .xml rules in the CSS
+            "html"
           when "bash"
-            "shell-session"
+            "console"
+          when "plain"
+            "plaintext"
```

Comment:
Could this be ✂️, if it's the same as the default case?

Reply:
Fixed.